backend/routes/userRoutes.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const { registerUser, loginUser } = require("../controllers/userController");
+const router = express.Router();
+
+const loginRouter = express.Router();
+
+router.post("/", registerUser);
+router.post("/login", loginUser );
+
+// loginRouter.get("/", (req, res) => {
```

Comment:
I would also remove these comments here as well as they don't seem to be needed (Just to reduce the amount of comments that we have for this file)

Reply:
Fixed: Changes are reflected in recent commits